setup.php

```diff
@@ -92,6 +92,30 @@ public function enqueue_scripts($page_hook) {
     );
     wp_enqueue_script('yoast-cmb2-plugin-js');
   }
+
+
+  /**
+   * Test whether Yoast SEO is hidden either by choice of the admin or because
+   * the post type is not a public post type
+   *
+   *
+   * @param  string $post_type (optional) The post type to test, defaults to the current post post_type.
+   *
+   * @return  bool        Whether or not the yoast seo metabox should be hidden
+   */
+  function check_if_yoast_seo_is_hidden( $post_type = null ) {
+    if ( ! isset( $post_type ) && ( isset( $GLOBALS['post'] ) && ( is_object( $GLOBALS['post'] ) && isset( $GLOBALS['post']->post_type ) ) ) ) {
```

Comment:
For sure, it's your plugin, so you do what you think is best :-)

But I do really like the whitespaces as described in the WordPress coding standards:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage